wunderio/next4drupal-project
# Nex-30: Language swticher

*Link to ticket: [JIRA-123](https://wunder.atlassian.net/jira/software/c/projects/NEX/boards/389?modal=detail&selectedIssue=NEX-30)*

### Backend

No Changes
  
### Frontend

Refactors language switcher component

### How to test
Simply fetch the branch and run the application. Click on the top right navigation to toggle the language dropdown. Select the desired language. The page should update accordingly showing the translations. It fetches all the languages from backend. Currently it supports English, Finnish and Swedish

### Screenshots


<img width="1464" alt="Screenshot 2023-01-18 at 12 10 55" src="https://user-images.githubusercontent.com/12473188/213106863-cb52c795-62a0-4dfd-a666-21338bc3e5e7.png">

<img width="1464" alt="Screenshot 2023-01-18 at 12 10 42" src="https://user-images.githubusercontent.com/12473188/213106874-8f430d44-aac0-452a-ac08-e70cfdd68ac9.png">
<img width="1464" alt="Screenshot 2023-01-18 at 12 10 36" src="https://user-images.githubusercontent.com/12473188/213106901-55e81d0f-4ade-4d62-8154-88f44c202696.png">

<img width="1464" alt="Screenshot 2023-01-18 at 12 10 01" src="https://user-images.githubusercontent.com/12473188/213107035-fbfe95cd-b2da-4c4c-8587-4e2834e210fe.png">


[JIRA-123]: https://wunder.atlassian.net/browse/JIRA-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
